MindPXv2/README.md

```diff
@@ -0,0 +1,55 @@
+# MindPX V2 Hardware
```

Comment:
Placeholder - just copied from the old readme.  Updates will include note that this is a derived product and link to additional PX4 docs.